src/v2/Components/ArtistInsight.tsx

```diff
@@ -52,20 +53,37 @@ export class ArtistInsight extends React.Component<ArtistInsightProps> {
     this.setState({ expanded: true })
   }
 
+  getTextByTheme() {
```

Comment:
Never occurred to me we'd want to do these kind of switches in class components 😱